wagtail/images/models.py

```diff
@@ -918,15 +916,15 @@ def check(cls, **kwargs):
         return errors
 
     @staticmethod
-    def construct_cache_key(image_id, filter_cache_key, filter_spec):
-        return "image-{}-{}-{}".format(image_id, filter_cache_key, filter_spec)
+    def construct_cache_key(image, filter_cache_key, filter_spec):
+        return "image-{}-{}-{}".format(image.file_hash, filter_cache_key, filter_spec)
```

Comment:
Note to reader: This change will invalidate existing rendition caches. Do we want to put a note about this somewhere?

Reply:
I think that should be fine. We can mention it in the release notes line